src/http/server.ts

```diff
@@ -142,6 +156,7 @@ export default class Server {
     if (configs.static_paths) {
       this.directory = configs.directory; // blow up if this doesn't exist
       configs.static_paths.forEach(path => {
+        console.log(path)
```

Comment:
we love console logging, but this has to go away :)

Reply:
Noooo!! Legit can't believe I left it in :P What would be do without PRs/Code Reviews

Ill edit it now